submissions/igarok88/a-tiny-JS-world-OOP/index.js

```diff
@@ -0,0 +1,49 @@
+import { print } from "./js/lib.js";
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+
+   Code repository: https://github.com/igarok88/a-tiny-JS-world
+   Web app: https://igarok88.github.io/a-tiny-JS-world/
+   */
+
+class Inhabitants {
+  constructor(species, name, gender, legs, saying) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.saying = saying;
+  }
+
+  getInhabitantProps() {
+    return [this.species, this.name, this.gender, this.legs, this.saying];
```

Comment:
This method has a high chance to fail architecturally over time as it is developer for a very specific use case. However, the current implementation is all right.

To learn what I mean you could read _Clean Architecture_ by Robert Martin at some point in the future.